app/controllers/protocols_controller.rb

```diff
@@ -1116,7 +1116,7 @@ def move_protocol(action)
   end
 
   def set_inline_name_editing
-    return unless can_manage_protocol_in_repository?(@protocol)
+    return unless can_manage_protocol_name_in_repository?(@protocol)
```

Comment:
I wouldn't create a separate permission helper just for this one case.
Better create a method in protocol:
  def initial_draft?
    in_repository_draft? && parent.blank?
  end

and than add it before can_manage_protocol_in_repository? check